xbmc/video/ContextMenus.cpp

```diff
@@ -270,9 +269,10 @@ bool CVideoResume::IsVisible(const CFileItem& itemIn) const
 
 namespace
 {
-std::vector<std::string> GetPlayers(const CPlayerCoreFactory& playerCoreFactory,
-                                    const CFileItem& item)
+bool HasMultiplePlayers(const CFileItem& item)
```

Comment:
This function is mostly identical to `GetPlayers` in VideoPlayActionProcessor.cpp. Maybe move `GetPlayers` to VideoUtils.h/cpp and call it here inside `HasMultiplePlayers`?

Reply:
Well spotted. Thanks. I will change this.